.github/workflows/build.yml

```diff
@@ -95,8 +97,31 @@ jobs:
           cache: 'yarn'
       - name: Install dependencies with yarn
         run: yarn install --frozen-lockfile
+
+      - # The output of `xcode-select --print-path` is something like `/Applications/Xcode-16.3.0.app/Contents/Developer`
+        # We want to extract the path to the Xcode.app, which is `/Applications/Xcode-16.3.0.app`
+        name: Find path to Xcode.app
+        id: find-xcode-app-path
+        if: startsWith(matrix.os, 'macos-')
+        run: |
+          XCODE_APP_PATH=$(xcode-select --print-path | sed 's|/Contents/Developer||')
```

Comment:
@philipphofmann I am stuck here. For some reason it keeps using the wrong SDK directory:

Expected: "/Applications/Xcode_16.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk"
Received: "/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk"

Do you have any ideas?